setup/www/README.md

```diff
@@ -12,7 +12,7 @@ Host iojs-www
   User root
 ```
 
-This setup assumes a GitHub Webhook is configured ([here](https://github.com/iojs/iojs.github.io/settings/hooks)) to point to port `9999` of the server with the path `/webhook` plus an additional secret.
+This setup assumes a GitHub Webhook is configured ([here](https://github.com/iojs/website/settings/hooks)) to point to port `9999` of the server with the path `/webhook` plus an additional secret.
 
```

Comment:
Unrelated to my PR, but this webhook doesn't exist as far as I can tell. Can we tweak the README? (Not sure where this is supposed to be though.)